osu.Game/Configuration/OsuConfigManager.cs

```diff
@@ -37,6 +37,7 @@ protected override void InitialiseDefaults()
 
             Set(OsuConfig.ShowInterface, true);
             Set(OsuConfig.KeyOverlay, false);
+            Set(OsuConfig.UserHidden, false);

```

Comment:
i meant to only change this in the usage in the class, not the config file. now it has no context